### PR TITLE
qa, doc: Fix comment

### PR DESCRIPTION
**Pull request by hebasto**
This PR is a follow-up for:
- https://github.com/bitcoin/bitcoin/pull/9956
- https://github.com/bitcoin/bitcoin/pull/10096

### CODE REVIEW

**Comment by kevkevinpal** [comment]
Concept ACK [011c77d](https://github.com/bitcoin/bitcoin/pull/28181/commits/011c77dd8d109b6b4d1291a778fa9c7f85750c77)

**Review by maflcko — APPROVED** [review]

File: test/functional/test_runner.py [hunk 1]
```diff
@@ -784,7 +784,7 @@ def check_script_list(*, src_dir, fail_on_warn):
     """Check scripts directory.
 
     Check that there are no scripts in the functional tests directory which are
```
[the remark this answers is not in the document]
Comment (hebasto) [reply test/functional/test_runner.py]: Thanks! Updated.

File: test/functional/test_runner.py [hunk 2]
```diff
@@ -783,8 +783,8 @@ def check_script_prefixes():
 def check_script_list(*, src_dir, fail_on_warn):
     """Check scripts directory.
 
-    Check that there are no scripts in the functional tests directory which are
-    not being run by pull-tester.py."""
+    Check that there all python files in this directory are categorized
```
Comment (fanquake) [inline test/functional/test_runner.py:786]:
```suggestion
    Check that all python files in this directory are categorized
```

Comment (hebasto) [reply test/functional/test_runner.py]: Done.

**Comment by RandyMcMillan** [comment]
ACK ab498d913c6f9f6096c75cc43a91e7a12cfc3fb7

**Pull request merged**